superset-frontend/src/views/CRUD/data/query/QueryList.tsx

```diff
@@ -0,0 +1,356 @@
+                'Duration: %s',
+                moment
+                  .utc(moment(endUtc).diff(startMoment))
+                  .format('HH:mm:ss.SSS'),
+              )}
+              placement="bottom"
+            >
+              <span>{formatedStartTime}</span>
+            </Tooltip>
+          );
+        },
+      },
+      {
+        accessor: 'tab_name',
+        Header: t('Tab Name'),
+      },
+      {
+        accessor: 'database.database_name',
+        Header: t('Database'),
+        size: 'xl',
+      },
+      {
+        accessor: 'schema',
+        Header: t('Schema'),
+        size: 'xl',
+      },
+      {
+        Cell: ({
+          row: {
+            original: { sql_tables: tables },
+          },
+        }: any) => {
+          const names = tables.map((table: any) => table.table);
+          const main = names.length > 0 ? names.shift() : '';
+
+          if (names.length) {
+            return (
+              <StyledTableLabel>
+                <span>{main}</span>
+                <Popover
+                  placement="right"
+                  title={t('TABLES')}
+                  trigger="click"
+                  content={
+                    <>
+                      {names.map((name: string) => (
+                        <StyledPopoverItem>{name}</StyledPopoverItem>
+                      ))}
+                    </>
+                  }
+                >
+                  <span className="count">(+{names.length})</span>
+                </Popover>
+              </StyledTableLabel>
+            );
+          }
+
+          return main;
+        },
+        accessor: 'sql_tables',
+        Header: t('Tables'),
+        size: 'xl',
+        disableSortBy: true,
+      },
+      {
+        accessor: 'user.first_name',
+        Header: t('User'),
+        size: 'xl',
+        Cell: ({
+          row: {
+            original: { user },
+          },
+        }: any) => `${user.first_name} ${user.last_name}`,
+      },
+      {
+        accessor: 'rows',
+        Header: t('Rows'),
+        size: 'lg',
+      },
+      {
+        accessor: 'sql',
+        Header: t('SQL'),
+        Cell: ({
+          row: {
+            original: { sql },
+          },
+        }: any) => {
+          return (
+            <StyledSyntaxHighlighter language="sql" style={github}>
+              {shortenSQL(sql)}
+            </StyledSyntaxHighlighter>
+          );
+        },
+      },
+      {
+        Header: t('Actions'),
+        id: 'actions',
+        disableSortBy: true,
+        Cell: ({
+          row: {
+            original: { id },
+          },
+        }: any) => {
+          return (
+            <Tooltip title={t('Open query in SQL Lab')} placement="bottom">
+              <a href={`/superset/sqllab?queryId=${id}`}>
+                <Icon name="full" />
+              </a>
+            </Tooltip>
+          );
+        },
+      },
+    ],
+    [],
+  );
+
+  const filters: Filters = useMemo(() => [], []);
```

Comment:
why do we need `useMemo` for this? if it's always a function that returns an array, why not make that a const outside of the render function and them use it in `TopAlignedListView`?

Reply:
It's currently an empty array at the moment, but will soon be an array with dynamic values as it is in other list views, eg https://github.com/preset-io/incubator-superset/blob/091432ea8ea4ddd7281eaf41c261f006fe99b352/superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx#L319